ui/pages/confirmations/components/simulation-details/useSimulationMetrics.ts

```diff
@@ -77,6 +80,16 @@ export function useSimulationMetrics({
 
   useIncompleteAssetEvent(balanceChanges, displayNamesByAddress);
 
+  // Skip metrics if simulation disabled or chain not supported.
+  if (
+    [
```

Comment:
Should we make this array an exported const somewhere?

Reply:
I considered it, but this and the list in `SimulationDetails` for hiding are ultimately distinct checks as one is saying what errors should hide the component, where as this is what error states result in metrics we don't want.

Hence why I thought it would be more explicit to do those checks inline without a constant.